src/Umbraco.Core/Services/MemberTypeService.cs

```diff
@@ -73,14 +73,15 @@ public class MemberTypeService : ContentTypeServiceBase<IMemberTypeRepository, I
 
     public string GetDefault()
     {
-        using (ICoreScope scope = ScopeProvider.CreateCoreScope(autoComplete: true))
+        using (ICoreScope scope = ScopeProvider.CreateCoreScope())
         {
             scope.ReadLock(ReadLockIds);
 
             using (IEnumerator<IMemberType> e = _memberTypeRepository.GetMany(new int[0]).GetEnumerator())
             {
                 if (e.MoveNext() == false)
                 {
+                    scope.Complete();
```

Comment:
Again: we might want to rewrite or remove the `Complete()` call before throwing an exception.